shared/test/integration/helpers/pageRegion.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var View = require('./view');
+
+function PageRegion(client, element) {
+	View.call(this, client);
```

Comment:
Hmmm...not sure why this is showing as no indentation. But I fixed and pushed again.